opencti-platform/opencti-front/src/private/components/common/form/FormAuthorizedMembers.tsx

```diff
@@ -64,7 +67,9 @@ const FormAuthorizedMembers = ({
                 name="authorizedMembers"
                 component={AuthorizedMembersField}
                 owner={owner}
+                currentUser={me}
```

Comment:
why do we add currentUser here since it's not used afterwards in AuthorizedMembersField ? we only use 'me'. Maybe we don't need at all this param